src/OpenApi/Schema.hs

```diff
@@ -195,15 +195,20 @@ newtype Enum = Enum [JSON.Value]
   deriving newtype (JSON.FromJSON, JSON.ToJSON)
   deriving stock   (Eq, Show)
 
-data Format = UnixTime
-  deriving stock (Eq, GHC.Bounded, GHC.Enum, Show)
+data Format
+  = CustomFormat Text  -- Non standard formats, explicitly allowed by OAS.
+  | UnixTime
+  deriving stock (Eq, Show)
```

Comment:
I noticed the `GHC.Bounded` and `GHC.Enum` instances were dropped too. Should this removal be extracted into a separate commit? I assume nothing in this project depends on it, otherwise the code would not compile.

Reply:
No, as the reason we remove these constraints lie in the addition of the `CustomFormat` constructor that is not enumeratable as it takes any `Text`.